python/python/glide/async_commands/standalone_commands.py

```diff
@@ -566,3 +566,72 @@ async def lolwut(
             str,
             await self._execute_command(RequestType.Lolwut, args),
         )
+
+    async def scan(
+        self,
+        cursor: int,
+        match: Optional[str] = None,
+        count: Optional[int] = None,
+        type: Optional[ObjectType] = None,
+    ) -> List[Union[int, List[str]]]:
+        """
+        Incrementally iterate over a collection of keys.
+        SCAN is a cursor based iterator. This means that at every call of the command,
```

Comment:
same - simplify the doc

Reply:
Understood about the cluster scan function. Altho when needed to help clients with problem in the libraries they use with cluster scan, some technichal details would help me or the client understand wired behaviours.  What here is too detailed?

Reply:
Done